pkgs/applications/networking/cluster/kubectl-klock/default.nix

```
@@ -1,17 +1,23 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGo121Module, fetchFromGitHub, makeWrapper }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub, makeWrapper }:
```

go_1_21 is default, also in future overrides need to be done in all-packages.nix as per https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults